kolibri/core/auth/management/commands/exportusers.py

```diff
@@ -57,15 +56,13 @@ def handle_async(self, *args, **options):
         else:
             filename = options["output_file"]
 
-        filepath = os.path.join(os.getcwd(), filename)
```

Comment:
There is a slight issue with making this change universally in the management commands.

Our documentation has explicit instruction on using these commands to directly output a file to the current working directory, and read from the current working directory too: https://kolibri.readthedocs.io/en/latest/manage/command_line.html#import-and-export-user-data-from-a-csv-file

The completely clean approach would be separating out a lot of this functionality out of the management commands in order to allow this to be done cleanly - but in the short term, we might need to just have a way to bypass storage use and work directly with a file path to continue to support this?